go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/aws-controllers-k8s/sagemaker-controller
 go 1.14
 
 require (
-	github.com/aws-controllers-k8s/runtime v0.2.0
+	github.com/aws-controllers-k8s/runtime v0.2.1-0.20210527234121-a3e2184a6be7
```

Comment:
Since we plan to release this version, IMO we should point to a released runtime.

Reply:
yes, pinged ack team